File: Difficulty: Easy/Armstrong Numbers/armstrong-numbers.java
// User function Template for Java
class Solution {
    static boolean armstrongNumber(int n) {
        // code here
        int s=n;
        int sum=0;
        while(n>0){
            int digit=n%10;
            n=n/10;
            sum=sum+(int)Math.pow(digit,3);
        }
        if(sum==s){
            return true;
        }
        return false;
    }
}

File: Difficulty: Easy/Armstrong Numbers/armstrong-numbers.py
class Solution:
    def armstrongNumber (self, n):
        # code here 
        num=n
        le=len(str(n))
        sum=0
        
        while n>0:
            d=n%10
            sum+=d**le
            n=n//10
        if sum==num:
            return True
        else:
            return False